### PR TITLE
Adding data protection 2.0 RA

### PR DESCRIPTION
**Pull request by manishjha86**
Signed-off-by: Manish Jha <er.manish.jha@gmail.com>

### CODE REVIEW

File: src/reference-designs/tko-data-protection.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## **Tanzu Kubernetes Grid Data Protection**
+
+Kubernetes maintains many states stored in the cluster itself in addition to your application state. Config maps, custom resource definitions, and secrets are all stored in the Kubernetes control plane. All of these are critical to your clusters’ operations. Protecting this data is just as critical as protecting any other element of your IT infrastructure, so it should be covered by data protection and your disaster recovery plan.
```
Comment (craigtracey) [inline src/reference-designs/tko-data-protection.md:3]: What about deployed application state?

File: src/reference-designs/tko-data-protection.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+- the entire backup
+- selected namespaces from the backup
+- specific resources from the backup identified by a given label
+
+You can also configure backup scheduling. You can specify how often to back up the Kubernetes data and how long to retain your backups. When the retention period for a backup expires, Tanzu Mission Control data protection automatically removes the expired backup from storage, lowering your overall storage usage.
+
+When it's time to destroy a cluster, Tanzu Mission Control gives you the choice of deleting your backup storage for that cluster or saving it for future data recoveries to a new cluster.
+
+When you enable data protection for a cluster, Tanzu Mission Control installs Velero with restic (an open-source backup tool), configured to use the opt-in approach. With this approach, you must annotate the pods whose volumes you want to include in backups. <_Is this info correct for the current version of Valero? Is it still necessary to annotate pods?_>
```
Comment (craigtracey) [inline src/reference-designs/tko-data-protection.md:26]: Can we address this question?

File: wordlist.txt [hunk 3]
```diff
@@ -60,3 +60,4 @@ VNET
 TKG
 NSXT
 autoscaler
+airgap
```
Comment (craigtracey) [inline wordlist.txt:63]: Please change to "air-gapped"

File: src/reference-designs/tko-data-protection.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- If a given pod is annotated to be backed up, persistent volume snapshots are included in the backup.
+- If a pod is not annotated to be backed up, restic does not include the persistent volume snapshots in the backup. However, Velero still creates snapshots if the specified target location for the backup is in the same cloud provider account as the cluster.
+
+Tanzu Mission Control manages the entire Valero lifecycle automatically, eliminating the need for you to manually update or upgrade it when newer software releases are available.
+
+If Tanzu Mission Control data protection does not handle all of your backup requirements, there is also a standalone version of Velero that you can install on your Tanzu Kubernetes clusters. Using the Velero standalone is advisable in the following scenarios:
+
+- In an internet-restricted or airgap environment, where the Tanzu Kubernetes clusters are not connected to the internet.
```
Comment (craigtracey) [inline src/reference-designs/tko-data-protection.md:35]: "air-gapped" please

**Review by craigtracey — CHANGES_REQUESTED** [review]

File: wordlist.txt (hunk 3 above)
Comment (srinivasme21) [reply wordlist.txt:63]: Craig, airgap without the hypen is good.

File: src/reference-designs/tko-data-protection.md (hunk 4 above)
Comment (srinivasme21) [reply src/reference-designs/tko-data-protection.md:35]: Craig, airgapped, without the hyphen is good.

**Review by srinivasme21 — APPROVED** [review]
LGTM

File: src/reference-designs/tko-data-protection.md (hunk 1 above)
Comment (manishjha86) [reply src/reference-designs/tko-data-protection.md:3]: The first line itself talks about the application state. Application state here refers to the deployed application itself.

File: src/reference-designs/tko-data-protection.md (hunk 2 above)
Comment (manishjha86) [reply src/reference-designs/tko-data-protection.md:26]: Yes annotation is needed for the pods

**Pull request merged**